spec/parliament/request/base_request_spec.rb

```diff
@@ -27,10 +27,10 @@
 
   describe '#get' do
     context 'it returns a status code of 200 and ..' do
-      subject { Parliament::Request::BaseRequest.new(base_url: 'http://localhost:3030/parties/current').get }
+      let(:base_response) { Parliament::Request::BaseRequest.new(base_url: 'http://localhost:3030/parties/current').get }
 
       it 'returns a Net::HTTPResponse' do
```

Comment:
This actually returns a BaseResponse right? Should there be an additional spec to cover the response method within the BaseResponse class? This should check that a BaseResponse is returned?